src/insns/load_tag_perms.adoc

```diff
@@ -0,0 +1,11 @@
+Resulting value of `cd`::
+The tag value written to `cd` is 0 if the tag of the memory location loaded is
+0 or the authorizing capability (<<ddc>> or `cs1`) does not grant <<c_perm>>.
++
+If the authorizing capability has both <<em_perm>> and <<w_perm>> unset, and the tag of `cd` is 1 and `cd` is not sealed, then an implicit <<ACPERM>> clearing <<w_perm>> and <<em_perm>> is performed to obtain the final permissions on `cd`.
+
+NOTE: Missing <<em_perm>> does not affect untagged values since this could result in surprising bit patterns when copying non-capability data.
+Similarly, sealed capabilities are not modified as they are not directly dereferenceable.
+
+NOTE: The implicit <<ACPERM>> does not introduce a new dependency on the loaded data since the actual masking of permissions can be deferred until the loaded capability is dereferenced or the metadata bits are inspected using <<GCPERM>> or <<GCHI>>.
+This is similar to the final value of the tag bit when the authorizing capability does not have <<c_perm>>.
```

Comment:
I disagree with this statement and would remove/modify it. The key difference is that the machine blindly clears the tag bit of the loaded cap when c_perm is clear; there is no need to decode/encode anything. However, em_perm does require decoding the permissions of the loaded cap, clearing some permissions, and encoding the permissions again which is not as straight forward because there are some restrictions on what permissions can be set/cleared and the RV32 permissions encoding is not a bitmap.

Reply:
Dropped this part.